target_chains/solana/programs/pyth-push-oracle/Cargo.toml

```diff
@@ -21,9 +21,9 @@ pythnet-sdk = { path = "../../../../pythnet/pythnet_sdk" }
 solana-program = { workspace = true }
 byteorder = "1.4.3"
 pyth-solana-receiver-sdk = { path = "../../pyth_solana_receiver_sdk"}
-pyth-solana-receiver = { path = "../pyth-solana-receiver", features = ["cpi"]}
```

Comment:
After this, a program that CPIs into pyth-solana-receiver doesn't need to have this dependency